app/src/main/java/com/keylesspalace/tusky/entity/Status.kt

```diff
@@ -120,45 +121,17 @@ data class Status(
     }
 
 
-    class Mention {
-        var id: String? = null
+    data class Mention (
```

Comment:
Why it was not even a data class before? Was there some problem with embedding somewhere?

Reply:
probably a mistake